services/efs-guardian/tests/unit/test_api_health.py

```diff
@@ -3,10 +3,14 @@
 # pylint:disable=redefined-outer-name
 
 import httpx
+from settings_library.rabbit import RabbitSettings
 from starlette import status
 
+pytest_simcore_core_services_selection = ["rabbit"]
+pytest_simcore_ops_services_selection = []
 
-async def test_healthcheck(client: httpx.AsyncClient):
+
+async def test_healthcheck(rabbit_service: RabbitSettings, client: httpx.AsyncClient):
     response = await client.get("/")
```

Comment:
for the healthcheck, I think if you look in autoscaling/clusters-keeper I make the healthcheck go bad when connection to dependencies is failing. and I think you need to add the rabbitmq issue as well there no?

Reply:
I will modify the health check mechanism in different PR (I need to think/discuss a bit about it)